docs/product-installation-guide/src/main/asciidoc/clustering-dm-install-proc.adoc

```diff
@@ -20,24 +20,24 @@ A {EAP} 7.1 cluster is configured and Elasticsearch and AMQ Broker 7.1 are insta
 <system-properties>
   <property name="org.uberfire.nio.git.dir" value="
       <NIOGIT_DIR>"/>
-  <property name="appformer-cluster" value="true"/>
-  <property name="appformer-jms-url" value="tcp://<MAIN_NODE_IP>:61616"/>
+  <property name="appformer-jms-connection-mode" value="REMOTE"/>
+  <property name="appformer-jms-url" value="tcp://<AMQ_NODE_IP>:61616"/>
   <property name="appformer-jms-username" value="<AMQ_BROKER_USER>"/>
   <property name="appformer-jms-password" value="<AMQ_BROKER_PASSWORD>"/>
   <property name="org.appformer.ext.metadata.index" value="elastic"/>
   <property name="org.appformer.ext.metadata.elastic.port" value="9300"/>
   <property name="org.appformer.ext.metadata.elastic.host" 
-      value="<MAIN_NODE_IP>"/>
+      value="<ELASTICSEARCH_NODE_IP>"/>
   <property name="org.appformer.ext.metadata.elastic.cluster" 
       value="kie-cluster"/>
   <property name="org.appformer.ext.metadata.elastic.retries" value="10"/>
 </system-properties>
 ----
-. On the node where Elasticseach and AMQ Broker are installed, in {EAP} create a user with the `admin` role and a username and password that matches the values of `<AMQ_BROKER_USER>` and `<AMQ_BROKER_PASSWORD>`:
+. On the nodes where Elasticseach and AMQ Broker are installed, in {EAP} create a user with the `admin` role and a username and password that matches the values of `<AMQ_BROKER_USER>` and `<AMQ_BROKER_PASSWORD>`:
 +
 [source,bash]
 ----
-$ <MAIN_NODE_EAP_HOME>/bin/./add-user.sh -a --user <AMQ_BROKER_USER> --password <AMQ_BROKER_USER> --role admin
+$ <EAP_HOME>/bin/add-user.sh -a --user <AMQ_BROKER_USER> --password <AMQ_BROKER_USER> --role admin
 ----
 
 .  To start {CENTRAL}, enter one of the following commands on each node of the cluster:
```

Comment:
I'd say "Enter one of the following commands on each node of the cluster to start Business Central:" This is consistent with another section on AMQ, and is more recommended practice in steps (avoid "To do something" as a step).